elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/ViewTest.java

```diff
@@ -46,8 +50,20 @@ public void testViewAttribute() {
                 .sorting(new SortingImpl(sortMap, PlayerStatsWithView.class, dictionary))
                 .build();
 
-        List<Object> results = StreamSupport.stream(engine.executeQuery(query).getData().spliterator(), false)
+        List<Object> results;
+        FutureTask<QueryResult> queryResult = engine.executeQuery(query);
+        queryResult.run();
+        try {
+            QueryResult result = queryResult.get();
+            results = StreamSupport.stream(result.getData().spliterator(), false)
                 .collect(Collectors.toList());
+        } catch (TransactionException e) {
```

Comment:
For UnitTests, you can simply have the test throw Exception and avoid all these try catch blocks.

Look at my prior comments - all of this can be changed by just adding 4 characters - without these large blocks of duplicate code.